src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -151,6 +151,7 @@
             <class name="com.wikia.webdriver.testcases.mercurytests.old.widgettests.WeiboTests"/>
             <class name="com.wikia.webdriver.testcases.mercurytests.old.widgettests.GoogleFormTests"/>
             <class name="com.wikia.webdriver.testcases.mercurytests.old.RedirectionTests"/>
+            <class name="com.wikia.webdriver.testcases.mercurytests.CategoryTest"/>
```

Comment:
The test functionality should be indicated on this level. Form what the test does I can see it's
Navigating (it expects some url change).